smart_card_connector_app/src/chrome-api-provider.js

```diff
@@ -0,0 +1,62 @@
+goog.provide('GoogleSmartCard.ChromeAPIProvider');
+
+goog.require('GoogleSmartCard.PcscLiteServerClientsManagement.ServerRequestHandler');
+
+goog.scope(function() {
+
+const GSC = GoogleSmartCard;
+
+/**
+ * @constructor
+ * @extends goog.Disposable
+ */
+GSC.ChromeAPIProvider = function(serverMessageChannel, serverReadinessTracker) {
+  this.serverRequester_ =
+      GSC.PcscLiteServerClientsManagement.ServerRequestHandler(
+          serverMessageChannel, serverReadinessTracker, 'chrome');
+
+  chrome.smartCardProviderPrivate.onEstablishContextRequested.addListener(
```

Comment:
Somewhere we need to check for presence of the `smartCardProviderPrivate` property, because the API doesn't exist on old ChromeOS versions (also, AFAICS, it's still Dev-only even on current versions).